app/views/items/show.html.erb

```diff
@@ -29,3 +29,54 @@
     </div>
   </div>
 </div>
+
+<div class="row">
+<% if current_user %>
+    <div class="col s2"><p></p></div>
+    <div class="col s8">
+      <%= render partial: '/reviews/form' %>
+    </div>
+<% else %>
+  <div class="col s2"><p></p></div>
+  <div class="col s8">
+    <h5 id='guest-message'>Login or Create Account to Write a Review</h5>
+    <div class="row guest-links">
+        <%= button_to "Login", login_path, method: :get, class: "waves-effect waves-light btn login-to-review" %>
+        <%= button_to "Create Account", new_user_path, method: :get, class: "waves-effect waves-light btn create-acct-to-review"%>
+    </div>
+  </div>
+<% end %>
+</div>
+
+<% unless @reviews.empty? && @item.reviews.empty?%>
```

Comment:
I don't see a way to refactor at the moment, and I don't think it's very much logic. There's a lot happening in the view, and we could make a partial for the reviews if we want. I don't know that it would do much in the way of cleaner code, though.